### DISCUSSION TITLE
Video cuts by rearranging srt subtittles

### DISCUSSION THREAD

**Discussion by pianomanedit** [SubtitleEdit/subtitleedit#6507] [Ideas]
Hi,

I tried many way to do this but can't find any good solution. 

I work with many hours of video footage of interviews. I used Whisper  in Subtitle Edit to create subtitles description in SRT format.
Than I upload this to Premiere project where I add subtitles to video footage (very useful for fast searching by typing specyfic word)

So basicly my idea is that I can send SRT text to director, who is deleteting some lines of text and sending me back the file (lines of text with timecodes). Than I'd like to transfer Time Codes of subtitles to video cuts.

Problem starts when he changes order of subs (for exaple moving part from 2 minute to the begining etc).

So SRT file looks like this:

41
00:01:52,000 --> 00:01:55,000
text-text-text

45
00:02:03,000 --> 00:02:06,520
text-text-text

20
00:00:40,000 --> 00:00:43,000
text-text-text

21
00:00:43,000 --> 00:00:47,000
text-text-text

I tried to import this Srt to Subtitle Edit, save timecodes as text, 
and insert new time stamps (based on subtitle duration) starting from 00:00:00:00. 

Than I'd like to export file as a EDL (edit dessision list) or XML to premiere and connect to video footage.
This way I could have edited parts of video (with changed order) done only by editing text inside SRT.

I can't find any solution to reset Timecodes in Subtitle Edit.

I know it looks a bit complicated but it would make the job a lot easier.
Maybe you have some ideas how to do this, or you know some other method to "text2videoedit"

Thanks

Simon

**Comment by darnn** [comment]
I have no idea about any of the other steps, but I think Tools->Sort by->Start time and Tools->Renumber might help you.

**Reply by pianomanedit** [reply]
Thanks. I tried this before but the Idea is that subtitles are in diffrent order and I just reset the start and end time - based on duration (so for exaple some line from the middle is first now with new start and end time)

**Comment by darnn** [comment]
Then I'm not sure what you mean by resetting the timecodes. You've give the example of what the file you get looks like, can you give an example of the file you want? Like, are you talking about importing timecodes from a different file? If so, you can do that through File->Import->Time codes (provided that you have an actual timed subtitle file with the proper timecodes).
